src/service.ts

```diff
@@ -18,13 +18,13 @@
  * @module common/service
  */
 
-import * as arrify from 'arrify';
+import arrify from 'arrify';
 import {Duplexify} from 'duplexify';
-import * as extend from 'extend';
+import extend from 'extend';
 import {GoogleAuth} from 'google-auth-library';
-import * as is from 'is';
-import * as pify from 'pify';
-import * as r from 'request';
+import is from 'is';
```

Comment:
not a comment for this PR, but if possible, we should switch to `@sindresorhus/is`. I find that `is` has non-intuitive behavior, e.g. `is.object(new Error())` returns false.

Reply:
:nods: I opened #163 to track